moveit_core/collision_detection/src/collision_env.cpp

```diff
@@ -42,7 +42,7 @@
 // Logger
 static const rclcpp::Logger LOGGER = rclcpp::get_logger("moveit_collision_detection.collision_robot");
 
-static inline bool validateScale(double scale)
+static inline bool validateScale(const double scale)
```

Comment:
@griswaldbrooks helped me find this: https://abseil.io/tips/109

and this: https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#exception-48

The const here is modifying a definition so it has meaning and is a good change. This looked weird to me but after reading those links I'm on board with this change. I have never met a local variable I didn't want to make const.